infra/jupyterhub/templates/tljh_config.yml.j2

```diff
@@ -26,3 +26,5 @@ services:
   cull:
     timeout: 6000
     every: 600
+limits:
+  memory: 7.8G
```

Comment:
C'est pas un peu trop comme valeur ?

Reply:
on a 8GB sur la machine. 
Je veux juste éviter qu'une personne ne fasse crasher la machine tout seul sans le brider sur la mémoire dont il dispose. "Even if you want individual users to use as much memory as possible, it is still good practice to set a memory limit of 80-90% of total physical memory." http://tljh.jupyter.org/en/latest/topic/tljh-config.html (j'avoue, j'en ai mis plus que 90%). Tu dirais combien ? Sachant qu'on va manipuler des données de la base et que ce n'est pas tout léger.

Reply:
Je dirais 7Go max pour que les autres services puisses tourner quand tranquillement :)
Sinon est-ce possible de mettre une variable d'environnement ?